environment.yml

```
@@ -0,0 +1,7 @@
+name: snakemake-template
+channels:
+  - bioconda
+  - conda-forge
+dependencies:
+  # Dependencies for running scripts and tasks
+  - bioconda::snakemake-minimal>=8.26.0
```

Same comments I just made on #27 - include a pinned Python, and pin the Snakemake version

```suggestion
  # Base dependencies - Python and Snakemake
  - conda-forge::python=3.11.11
  - bioconda::snakemake-minimal=8.26.0
```